.github/workflows/vast.yaml

```diff
@@ -1167,21 +1167,38 @@ jobs:
         run: |
           poetry config repositories.test-pypi https://test.pypi.org/legacy/
           poetry config pypi-token.test-pypi "${{ secrets.TEST_PYPI_TOKEN }}"
+      - name: Publish ${{ needs.configure.outputs.release-version }} to Test PyPI
+        working-directory: python
+        run: |
+          poetry publish --build -r test-pypi --no-interaction
```

Comment:
Instead of adding one additional job for this, can we just add these two steps to the `python` job right above?

Reply:
good point, we could actually also merge the release job, couldn't we?

Reply:
I think that's cleaner to have separate given the on release condition, just to make sure there's no lingering debug build config that we package.